test/test_vmap.py

```diff
@@ -2106,16 +2106,41 @@ def test_slice(self):
              (torch.rand(3, 5, B0, B1, B2),), in_dims=2)
 
     def test_squeeze(self):
+        def verify_behavior(op, min_size=1):
+            test = self._vmap_view_test
+            B0, B1 = 1, 11
+            # These tests cannot be used with an operator that requires more
+            # than 1 dimension after batching.
+            if min_size <= 1:
+                test(op, (torch.rand(B0),))
+                test(op, (torch.rand(B1),))
+                test(vmap(op), (torch.rand(B0, B1, 1),))
+                test(vmap(op), (torch.rand(B1, 1, B0),), in_dims=2)
+            test(op, (torch.rand(B0, 3, 5),))
+            test(op, (torch.rand(1, B0, 5),), in_dims=1)
+            test(op, (torch.rand(B0, 0, 1, 5, 1),))
+            test(op, (torch.rand(B0, 1, 1, 1, 1),))
+            test(vmap(op), (torch.rand(B0, B1, 1, 3, 4),))
+            test(vmap(op), (torch.rand(B1, 1, B0, 4, 5),), in_dims=2)
+
+        verify_behavior(torch.squeeze)
+        verify_behavior(lambda x: torch.squeeze(x, dim=1), min_size=2)
+        verify_behavior(lambda x: torch.squeeze(x, dim=-1), min_size=2)
+        verify_behavior(lambda x: torch.squeeze(x, dim=-2), min_size=3)
+
```

Comment:
I think we can uncomment out https://github.com/facebookresearch/functorch/blob/b73213bad5c5c6aa5eeaa96a048bf1163f4cdb40/test/test_vmap.py#L2908. PyTorch has an operator database and we have some autogenerated tests for each operator in that database. Our autogenerated tests for vmap aren't as comprehensive as manually writing tests yet (so the tests you wrote are still helpful!)